data/json/items/gun/faults_gun.json

```diff
@@ -21,11 +21,11 @@
     "requirements": {  }
   },
   {
-    "id": "fault_gun_clogged",
```

Comment:
What happens if an old save containing clogged guns is loaded? Does the fault disappear without notice or will error messages show up?

Reply:
Not sure. How many people actually have saves with clogged guns?
Does anyone have a save with that condition? Doesn't seem very easy to get it since BP ammo is rare and since the condition is currently chance-based.

If anyone has a save like that, I will test. What's the usual procedure for updates that get rid of obsolete conditions? Do we just keep them on record indefinitely?

Reply:
You can toggle faults by gaining debug mutations and `m`end a gun in the item action menu (`i`nventory - select the gun - `m`end).

Reply:
OK, will try when I'm back. Hopefully the game just throws a debug msg and clears out the obsolete fault. I imagine the devs would have implemented such a feature since I can only assume they've had the issue of obsolete item conditions in the past.

Reply:
OK I put it into obsolete so I think it will work now.